### ISSUE TITLE
iOS15+ support keyboardLayoutGuide

### ISSUE THREAD

**Issue by KYHyeon** [s4cha/KeyboardLayoutGuide#38]
It appears to officially support keyboardLayoutGuide from iOS15+.
view.keyboardLayoutGuide causes errors **'keyboardLayoutGuide' is only available in iOS 15.0 or newer**
![image](https://user-images.githubusercontent.com/16751025/121221998-1a467300-c8c1-11eb-954e-9d2b4747fab6.png)

Reference: https://developer.apple.com/documentation/uikit/uiview/3752221-keyboardlayoutguide

**Comment by shgew** [comment]
If you import KeyboardLayoutGuide at the top of the file, the error goes away.

**Comment by s4cha** [comment]
@KYHyeon Yeah I saw they announced `keyboardLayoutGuide` and I figured names would probably collide.
Looks like we got "Sherlocked" but I'm glad this is now becoming official, it's every maintainer goal !
Does @shgew solution fixes your issue? What version of Xcode are you using to be able to target iOS 15 ?

**Comment by MainasuK** [comment]
The workaround not fix the app target for iOS 15 only. The app still find and use the `keyboardLayoutGuide ` from UIKit but not the package.

https://github.com/MainasuK/KeyboardLayoutGuide/tree/fix/iOS15
Rename the guide is works for me. That change will break the other dependencies. So if you think that is fine I could make a PR.

**Comment by s4cha** [comment]
@MainasuK that is weird, just tried locally creating an iOS15+ only app and whenever I add `import KeyboardLayoutGuide` at the top of the file, it uses the library one and the Native one when removing the import (see screenshots attached)
<img width="788" alt="Screenshot 2022-01-17 at 17 52 50" src="https://user-images.githubusercontent.com/1812953/149809908-6ca571c5-bad4-4f86-b435-43237fd8be4a.png">
<img width="788" alt="Screenshot 2022-01-17 at 17 53 02" src="https://user-images.githubusercontent.com/1812953/149809924-db655e3c-35c1-4b0c-8b6f-3ae1795e8597.png">

**Comment by MainasuK** [comment]
@s4cha You can try push a view in navigation stack or present a modal then dismiss it. The native one get a bug in iOS 15.0 build in my case. 

The bug also described in that blog:
https://useyourloaf.com/blog/keyboard-layout-guide/

**Comment by azun-khang** [comment]
Thanks @MainasuK . I have the same issue with you. Luckily, your workaround/fix works for me.
